docs/workflows/remove_content.rst

```diff
@@ -0,0 +1,22 @@
+Remove Content
+==============
+
+
+The process of removing a content involves a couple of Pulp-archictetural issues.
```

Comment:
I would rephrase this slightly, they aren't so much architectural issues as architectural reasons.

> Removing content from Pulp entirely is slightly complicated for a number of architectural reasons.